static-instrumenter/agent-instrumenter/src/main/java/io/opentelemetry/contrib/staticinstrumenter/PostTransformer.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.staticinstrumenter;
+
+import java.lang.instrument.ClassFileTransformer;
+import java.security.ProtectionDomain;
+import java.util.Arrays;
+import javax.annotation.Nullable;
+
+public class PostTransformer implements ClassFileTransformer {
+  @Override
+  @Nullable
+  public byte[] transform(
+      ClassLoader loader,
+      String className,
+      Class<?> classBeingRedefined,
+      ProtectionDomain protectionDomain,
+      byte[] classfileBuffer) {
+
+    TransformedClass pre = Main.getInstance().getCurrentClass();
```

Comment:
I guess we don't know yet how these transformers will be used, but maybe in one of the future PRs it'll be possible to pass the state in the constructor instead of a static singleton?